extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -140,7 +140,7 @@ define('ext.wikia.adEngine.slotTweaker', [
 			}
 
 			log(['Slot ratio', aspectRatio], 'debug', logGroup);
-			providerContainer.style.paddingBottom = 100/aspectRatio + '%';
+			// providerContainer.style.paddingBottom = 100/aspectRatio + '%';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)